src/main/java/com/epam/ta/reportportal/core/integration/util/IntegrationService.java

```diff
@@ -26,42 +26,7 @@
  */
 public interface IntegrationService {
 
-	/**
-	 * Create {@link Integration} with {@link Integration#project == NULL}
-	 *
-	 * @param integrationTypeName {@link com.epam.ta.reportportal.entity.integration.IntegrationType#name}
-	 * @param integrationParams   {@link com.epam.ta.reportportal.entity.integration.IntegrationParams#params}
-	 * @return new {@link Integration}
-	 */
-	Integration createGlobalIntegration(String integrationTypeName, Map<String, Object> integrationParams);
+	Map<String, Object> retrieveIntegrationParams(Map<String, Object> integrationParams);
 
-	/**
-	 * Create {@link Integration} for {@link com.epam.ta.reportportal.entity.project.Project} with provided ID
-	 *
-	 * @param integrationTypeName {@link com.epam.ta.reportportal.entity.integration.IntegrationType#name}
-	 * @param projectDetails      {@link com.epam.ta.reportportal.commons.ReportPortalUser.ProjectDetails}
-	 * @param integrationParams   {@link com.epam.ta.reportportal.entity.integration.IntegrationParams#params}
-	 * @return new {@link Integration}
-	 */
-	Integration createProjectIntegration(String integrationTypeName, ReportPortalUser.ProjectDetails projectDetails,
-			Map<String, Object> integrationParams);
-
-	/**
-	 * Update {@link Integration} with {@link Integration#project == NULL}
-	 *
-	 * @param id                {@link Integration#id}
-	 * @param integrationParams {@link com.epam.ta.reportportal.entity.integration.IntegrationParams#params}
-	 * @return updated {@link Integration}
-	 */
-	Integration updateGlobalIntegration(Long id, Map<String, Object> integrationParams);
-
-	/**
-	 * Updated {@link Integration} for {@link com.epam.ta.reportportal.entity.project.Project} with provided ID
-	 *
-	 * @param id                {@link Integration#id}
-	 * @param projectDetails    {@link com.epam.ta.reportportal.commons.ReportPortalUser.ProjectDetails}
-	 * @param integrationParams {@link com.epam.ta.reportportal.entity.integration.IntegrationParams#params}
-	 * @return updated {@link Integration}
-	 */
-	Integration updateProjectIntegration(Long id, ReportPortalUser.ProjectDetails projectDetails, Map<String, Object> integrationParams);
+	boolean validateIntegration(Integration integration, ReportPortalUser.ProjectDetails projectDetails);
```

Comment:
Maybe this method can be split into two:
1) for global integration
2) for project integration
If we have project-/global-scoped integrations, we can split validation logic according to this difference, so if we want to validate global integration, we can just call appropriate method without providing 'NULL' for ProjectDetails argument